src/AppBundle/Tests/Command/RemoveImageCommandTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace AppBundle\Tests\Command;
+
+use AppBundle\Command\RemoveImageCommand;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+class RemoveImageCommandTest extends KernelTestCase
+{
+    public function testExecuteCommand()
```

Comment:
This test - test only that command will execute command without Exception.
You can write functional test:
 - Check that db does not have broken MediaObjects
 - Write broken MediaObject
 - Check
 - execute a command
 - Check
Also, after refactoring RemoveBrokenMediaObjectsCommand:execute and devide it to some separate methods you can unit tested protected methods.